packages/sdk/package.json

```diff
@@ -41,10 +41,12 @@
     "@azure/cosmos": "^3.3.6",
     "@conversationlearner/models": "^0.501.2",
     "@conversationlearner/ui": "^0.501.15",
+    "@types/lru-cache": "^5.1.0",
     "@types/supertest": "2.0.4",
     "async-file": "^2.0.2",
     "body-parser": "1.18.3",
     "botbuilder": "4.4.0",
```

Comment:
I would expect these versions of `botbuilder` to all match, all be `4.7.0`.  Does `botbuilder-lg` fully work with `4.4`?

Reply:
I'll need to defer this question to @coldplaying42

Reply:
There is only one public version 4.7.0 of botbuilder-lg, and botbuilder-lg depends on botbuilder-core pkg but not botbuilder pkg (notice that botbuilder also depends on botbuilder-core pkg ).